lib/assertions.js

```diff
@@ -798,7 +798,22 @@ module.exports = function (expect) {
 
     expect.addAssertion('<Error> to [exhaustively] satisfy <Error>', function (expect, subject, value) {
         expect(subject.constructor, 'to be', value.constructor);
-        expect(subject, 'to have properties', expect.argTypes[0].unwrap(value));
+
+        var unwrappedValue = expect.argTypes[0].unwrap(value);
+        expect.withError(function () {
+            expect(subject, 'to [exhaustively] satisfy', unwrappedValue);
```

Comment:
Missing `return`? (`to satisfy` can return a promise if it's an async assertion)